packages/material-ui/test/Form.test.tsx

```diff
@@ -106,4 +105,96 @@ describe("single fields", () => {
     const tree = renderer.create(<Form schema={schema} />).toJSON();
     expect(tree).toMatchSnapshot();
   });
-});
\ No newline at end of file
+  test("password field", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "password",
+    };
+    const tree = renderer
+      .create(<Form schema={schema} uiSchema={uiSchema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("textarea field", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "textarea",
+    };
+    const tree = renderer
+      .create(<Form schema={schema} uiSchema={uiSchema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("select field", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+      enum: ["foo", "bar"],
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("checkbox field", () => {
+    const schema: JSONSchema7 = {
+      type: "boolean",
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("checkbox field", () => {
+    const schema: JSONSchema7 = {
+      type: "boolean",
+    };
+    const tree = renderer.create(<Form schema={schema} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("checkboxes field", () => {
+    const schema: JSONSchema7 = {
+      type: "array",
+      items: {
+        type: "string",
+        enum: ["foo", "bar", "fuzz", "qux"],
+      },
+      uniqueItems: true,
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "checkboxes",
+    };
+    const tree = renderer
+      .create(<Form schema={schema} uiSchema={uiSchema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test("radio field", () => {
+    const schema: JSONSchema7 = {
+      type: "boolean",
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "radio",
+    };
+    const tree = renderer
+      .create(<Form schema={schema} uiSchema={uiSchema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  // There is a bug in the Material UI <Slider /> component that prevents this from working.
+  // Error: `TypeError: Cannot read property 'addEventListener' of null`
+  // From: https://github.com/mui-org/material-ui/blob/v4.5.2/packages/material-ui/src/Slider/Slider.js#L622
```

Comment:
Note that this does not appear to be a new bug in this branch as I am seeing the same thing if I copy this test over to master.